middleware/security/jwt/jwt.go

```diff
@@ -91,18 +208,62 @@ func New(validationKeys interface{}, validationFunc goa.Middleware, scheme *goa.
 
 			incomingToken := strings.Split(val, " ")[1]
 
+			jwtKeyName := req.Header.Get("jwtkeyname")
```

Comment:
Would it make sense to make the name of the header configurable? Maybe it's a field on the resolver? `KeyNameHeader` or something like that?